lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -173,8 +174,13 @@ function createBucket(authInfo, bucketName, headers,
         null, null, null, null, null, null, null, null,
         null, isNFSEnabled);
 
-    if (locationConstraint !== undefined) {
-        bucket.setLocationConstraint(locationConstraint);
+    const { locationConstraintStr, isIngestion } =
+        locationConstraint.split(zenkoSeparator);
+    if (locationConstraintStr !== undefined) {
+        bucket.setLocationConstraint(locationConstraintStr);
+    }
+    if (isIngestion !== undefined) {
```

Comment:
it would be safer to explicitly test that the suffix is `ingest`, otherwise a location constraint of `us-west-2:fortnite` or `my-location:do-not-ingest`  would still make it an ingest bucket